program/src/processor.rs

```diff
@@ -3735,7 +3778,7 @@ impl Processor {
 
         let mut liqor_ma =
             MangoAccount::load_mut_checked(liqor_mango_account_ai, program_id, mango_group_ai.key)?;
-        check_eq!(liqor_ai.key, &liqor_ma.owner, MangoErrorCode::InvalidOwner)?;
+        check_eq!(liqor_ai.key, &liqor_ma.owner, MangoErrorCode::InvalidOwner)?; // todo
```

Comment:
what does the todo mean? todo: consider allowing delegate?

Reply:
thanks, correct, fixed